packages/studio-ui/package.json

```diff
@@ -53,7 +54,7 @@
     "moment": "^2.29.1",
     "ms": "^2.1.3",
     "mustache": "^2.3.0",
-    "nanoid": "^2.0.3",
+    "nanoid": "^3.1.30",
```

Comment:
Can you make sure to follow [this upgrade guide](https://github.com/ai/nanoid/releases/tag/3.0.0) or stay on v2.